src/utils/request.py

```diff
@@ -95,13 +95,6 @@ def _make_request(
     if retry_if_failed:
         session.mount("http://", _adapter)
         session.mount("https://", _adapter)
-    headers = {

```

Comment:
what made you get rid of this? Just curious

Reply:
Torbox api wasnt responding to those headers very well. If a service needs them then lets implement them st that level?

Reply:
Sure. I wasn't sure how the curl's from that list were gonna behave too